python/idsse_common/idsse/common/path_builder.py

```diff
@@ -361,6 +361,9 @@ def parse_args(key: str, value: str, result: dict):
         for i, _dir in enumerate(dirs):
             res = re.search(r'{.*}', _dir)
             if res:
-                parse_args(res.group(), vals[i][res.span()[0]:], time_args)
+                try :
```

Comment:
I'll need to incorporate this exception handling in the new implementation of path builder, thanks for finding/fixing. Is there a test that exercises this code?

Reply:
My test was running it against the MRMS http feed so no.